app/controllers/services/service_instances_controller.rb

```diff
@@ -159,6 +160,7 @@ def delete(guid)
         return [HTTP::NO_CONTENT, nil]
       end
 
+      validate_not_shared_space_delete(service_instance)
```

Comment:
Same naming nit pick as in PR 1040. Copy and pasted from that PR :-)

In general, we prefer to use positive assertions rather than negatives (not, un, etc.) because it's easier for us to reason about. I propose we change this method name to something like `validate_shared_space_deleteable`.